Results/LGBScore.txt

```diff
@@ -0,0 +1,3 @@
+2021-04-12 14:13:51 

```

Comment:
I don't think these result files need to be checked in. Remove them and add a gitignore.

Reply:
The goal of these  results file are used to save all results, that means save all results that all of us run it. So if one of us run the model, it will save results and every time , they can check-in this for saving.